src/main/java/seedu/trackermon/storage/JsonFileManager.java

```diff
@@ -5,18 +5,18 @@
 import java.nio.file.Files;
 import java.nio.file.Path;
 import java.nio.file.StandardCopyOption;
-import javax.swing.JFileChooser;
-import javax.swing.UIManager;
-import javax.swing.UnsupportedLookAndFeelException;
-import javax.swing.filechooser.FileFilter;
+
+import javafx.stage.FileChooser;
 
 public class JsonFileManager {
 
     public static final int SUCCESS = 0;
     public static final int CANCEL = 1;
     public static final int ERROR = 2;
+    private static final String IMPORT_TITLE = "Choose a JSON file to load into Trackermon";
+    private static final String EXPORT_TITLE = "Choose a location to save Trackermon's data";
 
-    private JFileChooser jfc;
+    private FileChooser fileChooser;
```

Comment:
Great job finding a better library that works with MacOS! Will have to keep this in mind next time I try to implement a File Explorer!